Source/Blazorise/wwwroot/colorPicker.js

```diff
@@ -26,8 +26,8 @@ export function initialize(dotnetAdapter, element, elementId, options) {
 
             // Main components
             preview: true,
-            opacity: true,
-            hue: false,
+            opacity: options.showOpacitySlider === true ? true : false,
```

Comment:
isn't this a truthful expression on itself?

Prefer  `options.showOpacitySlider === true` over `options.showOpacitySlider === true ? true : false`

Reply:
same for the ones below

Reply:
> isn't this a truthful expression on itself?
> 
> Prefer `options.showOpacitySlider === true` over `options.showOpacitySlider === true ? true : false`

Not necessarily always. In JS the truthful value can be any thing that is not false, non-empty string, number higher than 0, etc. This also ensures that the result is always bool.